sktime/forecasting/compose/_column_ensemble.py

```diff
@@ -221,3 +221,22 @@ def _check_forecasters(self, y):
                 "One estimator per column required. Found %s" % len(indices)
             )
         return self.forecasters
+
+    @classmethod
+    def get_test_params(cls):
+        """Return testing parameter settings for the estimator.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
```

Comment:
Please add a dot here and at the end of the line 235

Reply:
Done! Thanks for the feedback.